src/main/java/software/amazon/cloudformation/AbstractWrapper.java

```diff
@@ -373,9 +373,17 @@ private void logUnhandledError(final String errorDescription,
             logUnhandledError(e.getMessage(), request, e);
             return ProgressEvent.defaultFailureHandler(e, e.getErrorCode());
         } catch (final AmazonServiceException | AwsServiceException e) {
-            publishExceptionMetric(request.getAction(), e, HandlerErrorCode.GeneralServiceException);
-            logUnhandledError("A downstream service error occurred", request, e);
-            return ProgressEvent.defaultFailureHandler(e, HandlerErrorCode.GeneralServiceException);
+            if ((e instanceof AwsServiceException && ((AwsServiceException) e).isThrottlingException()) ||
+                (e instanceof AmazonServiceException && ((AmazonServiceException) e).getErrorCode() != null
+                    && ((AmazonServiceException) e).getErrorCode().contains("Throttling"))) {
```

Comment:
`((AmazonServiceException) e).getErrorCode().contains("Throttling")))` is not enough. some throttling exception don't use `Throttling`.  there is a `RetryUtils.isThrottlingException(ex)` which should cover all the case for SDK v1.

Reply:
gotcha, let me change it